README.md

```diff
@@ -366,6 +366,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 * [clippy](https://crates.io/crates/clippy) — Rust lints [<img src="https://api.travis-ci.com/rust-lang/rust-clippy.svg?branch=master">](https://travis-ci.org/rust-lang/rust-clippy)
 * [clog-tool/clog-cli](https://github.com/clog-tool/clog-cli) — generates a changelog from git metadata ([conventional changelog](https://blog.thoughtram.io/announcements/tools/2014/09/18/announcing-clog-a-conventional-changelog-generator-for-the-rest-of-us.html)) [<img src="https://api.travis-ci.org/clog-tool/clog-cli.svg?branch=master">](https://travis-ci.org/clog-tool/clog-cli)
 * [dan-t/rusty-tags](https://github.com/dan-t/rusty-tags) — create ctags/etags for a cargo project and all of its dependencies [<img src="https://api.travis-ci.org/dan-t/rusty-tags.svg?branch=master">](https://travis-ci.org/dan-t/rusty-tags)
+* [dotenv-linter](https://github.com/mgrachev/dotenv-linter) - Linter for `.env` files ![CI](https://github.com/mgrachev/dotenv-linter/workflows/CI/badge.svg)
```

Comment:
Can you please add a link to the actions CI and also the badge should really have a branch specified (as per https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#example-using-the-branch-parameter) so we don't accidentally get the results for non-master branches.

Reply:
Done.